### PR TITLE
Release 0.4.0

### PR DESCRIPTION
**Pull request by alexjfisher**

### CODE REVIEW

File: REFERENCE.md [hunk 1]
```diff
@@ -382,103 +383,103 @@ Port the service should listen on. Module default is `undef` which means that po
 isn't added to chrony.conf, and chrony listens to the default ntp port 123 if
 `queryhosts` is used.
 
-Default value: `undef`
+Default value: ``undef``
```
Comment (bastelfreak) [inline REFERENCE.md:386]: some values have a single backtick, some have two? never saw that in the past in a REFERENCE.md

**Review by bastelfreak — APPROVED** [review]

File: REFERENCE.md (hunk 1 above)
Comment (alexjfisher) [reply REFERENCE.md:386]: Odd.  Seems to render ok though.

Comment (alexjfisher) [reply REFERENCE.md:386]: I guess related to https://github.com/puppetlabs/puppet-strings/pull/243

Comment (bastelfreak) [reply REFERENCE.md:386]: > prevent emoji from being inadvertantly rendered
lol

**Pull request merged**